src/volt/components/ProposalsList/index.js

```diff
@@ -1,19 +1,18 @@
-import React from "react";
+import React, { memo } from "react";
 import { ListContainer } from "./styles";
 import SingleProposal from "./SingleProposal";
 
 const ProposalsList = props => {
   const { list, toggle, favorites } = props;
-
   return (
     <ListContainer>
       {list &&
-        list.map(proposal => {
+        list.map((proposal, index) => {
           const { proposalId } = proposal;
           const favorite = favorites[proposalId];
           return (
             <SingleProposal
-              key={proposalId}
+              key={`${proposalId}${index}`}
```

Comment:
Having proposalId in key doesn't make any sense in this case. What's the problem with it?

Reply:
Not sure I following.
In this specific case we had multiple proposals with exacty the same id - empty string. React was throwing lots of warning in the console. So I added index to eliminate those.
Just as other stuff - it was written before your PR that fixed routing issues.